packages/sku/lib/program/commands/setup-hosts/setup-hosts.command.js

```diff
@@ -0,0 +1,14 @@
+const { Command } = require('commander');
+
+const setupHosts = new Command('setup-hosts');
+
+setupHosts
+  .description(
+    'Update your hosts file to point any configured hosts to your local machine.',
+  )
```

Comment:
Should we mention here that you'll need admin (`sudo`) permissions?

Reply:
Yeah we could mention that here. I'll see if I can add it in concisely.